app/api/chat/route.ts

```diff
@@ -5,13 +5,19 @@ const openai = new OpenAI({
   apiKey: process.env.OPENAI_API_KEY,
 });
 
+const localai = new OpenAI({
+  baseURL: `http://127.0.0.1:5000/v1`,
```

Comment:
We can also create an environment variable for this. Something like: `BASE_ULR=http://127.0.0.1:5000`